geode-core/src/main/java/org/apache/geode/internal/statistics/StripedStatisticsImpl.java

```diff
@@ -38,12 +38,16 @@ public StripedStatisticsImpl(StatisticsType type, String textId, long numericId,
     StatisticsTypeImpl realType = (StatisticsTypeImpl) type;
 
     this.intAdders =
-        Stream.generate(LongAdder::new).limit(realType.getIntStatCount()).toArray(LongAdder[]::new);
+        Stream.generate(LongAdder::new)
```

Comment:
Is this a reformat for the sake of personal format preference?

Reply:
I think it was me playing with stuff that was out of scope for our PoC, and accidentally committing it.